scripts/setup-centos9.sh

```diff
@@ -59,7 +59,7 @@ function install_velox_deps_from_dnf {
   dnf_install libevent-devel \
     openssl-devel re2-devel libzstd-devel lz4-devel double-conversion-devel \
     libdwarf-devel elfutils-libelf-devel curl-devel libicu-devel bison flex \
-    libsodium-devel zlib-devel
+    libsodium-devel zlib-devel go
```

Comment:
Are we supporting Spark query runner only on Centos9 and not Ubuntu, Macos?

Reply:
We are supporting them. This change is removed as I find the compilation in centos also works without it.